### DISCUSSION TITLE
is there a step by step walk through

### DISCUSSION THREAD

**Discussion by gigerpimp** [spartacus04/jext-reborn#348] [Q&A]
is there a step by step walk through not sure if I missed a step or if something is wrong  I cant seam to find the disc.json i have the resources pack installed right and added the correct info to the server settings but no custom disc load in

**Comment by spartacus04** [comment]
There's this tutorial [here](https://spartacus04.github.io/jext-reborn/documentation/quickstart/) on the documentation page, however if that's unclear feel free to ask any questions

**Comment by gigerpimp** [comment]
I did all that and it all looks good but when I go to use the disc there
isn't any available

**Reply by spartacus04** [reply]
Are you using the built-in resource pack host, or are you hosting from the server.properties?

**Comment by gigerpimp** [comment]
I maybe doing both I have copied it to the sever in the geyser-spigot packs
folder and updated the server properties with the dropbos and hash and used
the plugin to make the files I put both versions Java and bedrock and
clicked to upload I havent been able to get any custom disc
